client/disputes/style.scss

```diff
@@ -5,6 +5,16 @@
 	.woocommerce-table__item a.components-button {
 		display: inline-flex;
 	}
+	.due-soon {
+		display: flex;
```

Comment:
I've asked in Slack about justifying this `center` rather than `left`.

I'm happy either way, but we may want to change it if we hear back in this thread: p1689211256800869/1689210150.726079-slack-C03LA109BKM

Reply:
Do you think this PR is good to merge? Do we need to wait from design team or change is needed we can do it in a separate PR?

Reply:
Just received confirmation: left aligned. https://a8c.slack.com/archives/C03LA109BKM/p1689218022338659?thread_ts=1689210150.726079&cid=C03LA109BKM

Good to merge :shipit: